### PR TITLE
Matuschkalulu data file

### PR DESCRIPTION
**Pull request by Matuschkalulu**
Hey guys can comes review this file -> it is already connected to the params file (fyi).

I think it is fine

### CODE REVIEW

**Review by maly-phy — COMMENTED** [review]
what is mpimg in line 10?

**Review by maly-phy — COMMENTED** [review]
I checked the params file but found it empty. You could start adding some like the data path, data size, etc.

**Comment by Matuschkalulu** [comment]
Params file should not be empty was already pushed and merged to the master branch - have you pulleed it ?

File: traffic_signs_code/params.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import os
 
-IMG_unrecognizable_path = os.path.join(os.getcwd(),'..','raw_data','Train','0')
+IMG_unrecognizable_path = os.path.join(os.getcwd(),'raw_data','Train','0')
 IMG_recognizable_path = os.path.join(os.getcwd(),'..','raw_data','Train','1')
+IMG_path =  os.path.join(os.getcwd(),'raw_data','Train')
 
 IMG_WIDTH = 100
```
Comment (Matuschkalulu) [inline traffic_signs_code/params.py:7]: IMG_WIDTH_BASE and IMG_WIFHT VGG

**Pull request merged**